pyabc/__init__.py

```diff
@@ -26,6 +26,8 @@
     SimpleFunctionDistance,
     PNormDistance,
     AdaptivePNormDistance,
+    AggregatedDistance,
+    AdaptiveAggregatedDistance,
```

Comment:
I don't know, but can't we combine both (AggregatedDistance and AdaptiveAggregatedDistance) functions in one and then use the adaptive flag to decide?

Reply:
Good point. Both have their advantages. I wanted to separate both to have clearly all basic functionality for weighting etc. in the base class, and all "adapting" functionality in the subclass, like for (Adaptive)PNormDistance. Note that ``adaptive`` means whether to only pre-calibrate the weights, or adapt them after each iteration. The name of the class might be a bit misleading, but I hope that's fine.